OutOfSchool/OutOfSchool.DataAccess/Models/BirthCertificate.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text;
+
+namespace OutOfSchool.Services.Models
+{
+    public class BirthCertificate
+    {
+        [Key]
+        [ForeignKey("Child")]
+        public long Id { get; set; }
+
+        /// <summary>
+        /// Gets or sets series of birth certificate.
+        /// </summary>
+        public string SvidSer { get; set; }
```

Comment:
Is it better to name it 'Series'?

then when you access it it will look like certificate.Series